src/assertions/CommonAssertions.ts

```diff
@@ -55,8 +55,8 @@ export class CommonAssertions {
     }
   }
 
-  public isEqualString(string1: string, string2: string, errorMessage: string) {
-    if (string1 !== string2) {
+  public isEqual(object1: any, object2: any, errorMessage: string) {
```

Comment:
Create a new assertion called `isEqualBigNumber` that uses the bigNumber equality check.